hazelcast-sql/src/main/java/com/hazelcast/jet/sql/impl/connector/infoschema/DataLinksTable.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright 2023 Hazelcast Inc.
+ *
+ * Licensed under the Hazelcast Community License (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://hazelcast.com/hazelcast-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.jet.sql.impl.connector.infoschema;
+
+import com.hazelcast.sql.impl.schema.ConstantTableStatistics;
+import com.hazelcast.sql.impl.schema.TableField;
+import com.hazelcast.sql.impl.schema.datalink.DataLink;
+import com.hazelcast.sql.impl.type.QueryDataType;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+
+import static java.util.Arrays.asList;
+
+public class DataLinksTable extends InfoSchemaTable {
+    private static final String NAME = "datalinks";
```

Comment:
should be added to `TableResolverImpl.ADDITIONAL_TABLE_PRODUCERS`

Reply:
Actually, it is added in `DataLinksResolver`, since it should be in the separate schema in the future.